### DISCUSSION TITLE
Couldn't find a compatible Webview2 Runtime installation to host WebViews.

### DISCUSSION THREAD

**Discussion by starkdm** [MicrosoftEdge/WebView2Feedback#3995] [Q&A]
Why am I getting this exception on Windows Server 2022, while my work computer (Windows 10) has the same WebView2 Runtime version installed and everything works fine?

EnsureCoreWebView2Async throws:
Microsoft.Web.WebView2.Core.WebView2RuntimeNotFoundException: Couldn't find a compatible Webview2 Runtime installation to host WebViews.
InnerException: System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)

Installed on the server from [here](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section) (both x64 x86)

![image](https://user-images.githubusercontent.com/50451195/196062753-1be0c825-9bab-45d5-b6a1-8da784c799c9.png)

**Comment by yunate** [comment]
Hi @starkdm!
Thanks for your reply. Could you help to test if the WPF sample[WPF sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WpfBrowser) could run ok in your Windows Server 2022 computer and check if the .Net SDK version is corresponding to the run time version?

**Comment by starkdm** [comment]
@yunate WPF Sample is targeting to .NET Core 3.1. I am target my project to .NET Core 6, but I am download latest WebView2 NuGet package version. In WPF Sample I see 1.0.1414.

**Comment by bbronisz** [comment]
I had similar issue and the solution was to run the installer as administrator so it is installed machine-wide. Otherwise it was installed for user that run the installer and when running a process as different user I was getting this exception.

**Reply by Renakos** [reply]
I have the issue now; running the installer to install machine-wide did not help.

**Reply by bbronisz** [reply]
@Renakos what is the exception you're getting? Could you paste the whole error + sample code?
